src/Framework/NativeMethods.cs

```diff
@@ -1537,11 +1537,11 @@ internal static (bool acceptAnsiColorCodes, bool outputIsScreen, uint? originalC
         return (acceptAnsiColorCodes, outputIsScreen, originalConsoleMode);
     }
 
-    internal static void RestoreConsoleMode(uint? originalConsoleMode)
+    internal static void RestoreConsoleMode(uint? originalConsoleMode, int outputStreamHandle)
```

Comment:
int --> enum (e.g. `internal enum StreamHandleType { StdOut = -11; StdErr = -12;}` and `internal static void RestoreConsoleMode(uint? originalConsoleMode, StreamHandleType streamHandleType )`)